prefetch.bs

```diff
@@ -863,6 +863,16 @@ The \`<code><dfn http-header>Sec-Speculation-Tags</dfn></code>\` HTTP request he
 
 The header field is an [[RFC9651]] Structured Header whose value must be a [=structured header/List=]. The list can contain either [=structured header/Token=] or [=structured header/String=] values. String values represent developer-provided tags, whereas token values represent predefined tags. As of now, the only predefined tag is `null`, which indicates a speculative navigation request with no developer-defined tag.
 
+<h2 id="interaction-with-other-specs">Interaction with other specifications and concepts</h2>
+
+<div algorithm="clear prefetch cache for origin patch">
+  Modify [$clear cache for origin$] to add the following step at the end.
+
+  1. [=list/For each=] [=prefetch record=] |prefetchRecord| in the user agent's |activeDocument|'s [=Document/prefetch records=]:
```

Comment:
|activeDocument| is not defined anywhere, and it doesn't sound like something the user agent has.

I think that, like https://wicg.github.io/nav-speculation/prerendering.html#interaction-with-clear-site-data-clear-cache , we should start from the user agent's top-level traversable set. From there we can get the [inclusive descendant navigables](https://html.spec.whatwg.org/#inclusive-descendant-navigables) and look at each of their `[=navigable/active document=]`s.

Reply:
Taking a look at how [inclusive descendant navigables ](https://html.spec.whatwg.org/#inclusive-descendant-navigables) is described as being "inclusive ancestor navigables of a Document", I'm a bit confused when you mean getting all the active documents from each inclusive descendant navigable.  My understanding was that for each traversable, there is one active document, with each active document potentially having multiple inclusive descendent navigables (line above the link)

Reply:
Yes, your understanding is basically correct. So the route to get all documents in the entire user agent would be something like:

```js
const topLevelTraversables = userAgent.topLevelTraversableSet();
const allNavigables = topLevelTraversables.flatMap(t => inclusiveDescendantNavigables(t.activeDocument));
const allDocuments = allNavigables.map(n => n.activeDocument);
```

(Expressing this in specs probably looks like a version of this that only uses `for`-`of` instead of `flatMap()`/`map()`.)

Reply:
Thanks! Just pushed with wording that was based on other HTML specs. Not sure if it would be better to have 3 for-loop statements here, so defined a list instead for one of them as it seems easier to read for me.